packages/ui-kit/src/components/PieChart/PieChart.stories.tsx

```diff
@@ -314,3 +315,21 @@ export const ThemeStory: Story = {
   ],
   render: (args) => <PieChart {...args} />
 }
+
+export const EmptyPieStory: Story = {
+  name: 'Empty State',
+  args: {
+    query: {
+      ...connectedParams,
+      filters: [
+        {
+          column: 'tortilla_id',
```

Comment:
On my side, it fails with
```
{
    "data": {
        "leaderboard": null
    },
    "errors": [
        {
            "message": "Unknown column \"tortilla_id\"",
            "extensions": {
                "code": "UNKNOWN_COLUMN"
            }
        }
    ]
}
```

Shall we put it into env maybe?

Reply:
Probably you don't have that dimension in your metric, let me set as an env variable so you can use any other